src/applications/mhv-secure-messaging/components/ThreadList/ThreadListItem.jsx

```diff
@@ -236,11 +233,7 @@ const ThreadListItem = props => {
         </div>
         <Link
           data-dd-action-name="Link to Message Subject Details"
-          aria-label={`${
-            unreadMessages ? 'Unread message.' : ''
-          } Message subject: ${categoryLabel}: ${subject}, ${formattedDate()}. ${
-            hasAttachment ? ' Has attachment.' : ''
-          }`}
```

Comment:
may need to add a child element i.e. `<span class="sr-only">Unread message</span>` to preserve. unread status in aria read out